notebooks/kubeflow_pipelines/pipelines/solutions/kfp_pipeline_vertex_prebuilt.ipynb

```diff
@@ -195,19 +195,20 @@
     "import os\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    from retrieve_best_hptune_component import retrieve_best_hptune_result
</pre><p>Add a blank line separating own imports from 3rd party imports (pep8): i.e. a blank line above this line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I personally agree, but actually it is the one pre-commit prefers (no blank line). It keeps reverting when we add a blank line between them.